software.md

```diff
@@ -0,0 +1,31 @@
+# FOSS United Community Software Policy
+This policy is devised to guide, but not mandate, software choices within the 
+FOSS United Community.
+
+- Define what we (the FOSS United Community) wish to do. Ask ourselves 
+whether what we wish to do will help in full-fulling the community's goals.
+- Define your requirements for the software solution clearly. Clearly 
+differentiate between needs and wants. Needs are features that are crucial to 
+performing the task at hand and wants are niceties, things we can do without.
```

Comment:
Do you mean the absolute requirements and good-to-have?

Reply:
Yes.